spec/controllers/workflows_controller_spec.rb

```diff
@@ -30,7 +30,7 @@
 
 RSpec.describe WorkflowsController do
   let!(:role) do
-    build_stubbed(:role).tap do |r|
+    build_stubbed(:project_role).tap do |r|
```

Comment:
Unrelated but can be bundled
```suggestion
    build_stubbed(:project_role) do |r|
```